TEveMu2e.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#Runs the TEve Event Display for Mu2e
+
+echo "Running TEve Event Display for Mu2e..."
+
+fcl=$1
+art=$2
+numofevts=$3
+
+for var in "$@"
+  do
+	if [ "$var" == '-3DOnly' ]; then
+	    sed -i 's/show2D.*/show2D : false/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == '-2Dand3D' ]; then
+	    sed -i 's/show2D.*/show2D : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == '-DSOnly' ]; then
+	    sed -i 's/showDSOnly :.*/showDSOnly : true/' TEveEventDisplay/fcl/prolog.fcl
+	    sed -i 's/showCRV :.*/showCRV : false/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+	
+        if [ "$var" == '-accumulate' ]; then
+	    sed -i 's/accumulate.*/accumulate : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+	
+	if [ "$var" == '-CRVOnly' ]; then
+	    sed -i 's/showDSOnly :.*/showDSOnly : false/' TEveEventDisplay/fcl/prolog.fcl
+	    sed -i 's/showCRV :.*/showCRV : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == '-DSandCRV' ]; then
+	    sed -i 's/showDSOnly :.*/showDSOnly : true/' TEveEventDisplay/fcl/prolog.fcl
+	    sed -i 's/showCRV :.*/showCRV : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'hits' ]; then
+	    sed -i 's/addHits.*/addHits : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'clusters' ]; then
+	    sed -i 's/addClusters.*/addClusters : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'tracks' ]; then
+	    sed -i 's/addTracks.*/addTracks : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'crvhits' ]; then
+	    sed -i 's/addCrvHits.*/addCrvHits : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'cosmictracks' ]; then
+	    sed -i 's/addCosmicSeedFit.*/addCosmicSeedFit : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+
+	if [ "$var" == 'mctraj' ]; then
+	    sed -i 's/addMCTraj.*/addMCTraj : true/' TEveEventDisplay/fcl/prolog.fcl
+	fi
+done
```

Comment:
Perhaps you could generate a new FCL file based on the `prolog.fcl` in the current working directory, based on the prolog.fcl, then run on that, rather than modifying FCL inside Offline.

A problem I see with this is that this would only work if the user's current working directory was the Offline folder. Also if the user is sourcing a release from CVMFS the `sed` commands would fail because the mount is readonly

Reply:
```bash

cp ${MU2E_BASE_RELEASE}/TEveEventDisplay/fcl/prolog.fcl EDcfg.fcl

...
sed -i '...' EDcfg.fcl
...

mu2e -c EDcfg.fcl

```

Reply:
If `${MU2E_BASE_RELEASE}/bin` is added to the `PATH` when a user runs `source setup.sh`, then:
- moving this script to `bin/`
- renaming the file to `TEveMu2e`
- running `chmod +x TEveMu2e`
- then git add && committing the file

the script should be accessible from any working directory in a mu2e environment by running
```
TEveMu2e <options>
```

Reply:
Thanks Ryun, yes that's true. I had some similar ideas - this part of the infastructure is very much in the development stages and SED is probably a better bet!

Reply:
Ah ok, I see what you mean now - I'll investigate and get this part of the code in better shape

Reply:
One thing we could so is maybe remove the TEveMu2e.sh from this PR and develop a better interface for the next update we do. This was something aditi made and I thought it would be useful but now thinking about it there are better ways to do this. I'll see what the reviewers say but we can pull it out for now and add a new interface when we do our next PR

Reply:
I have enacted this version- tested and it worked will await full review to see whether its OK with Rob/Dave

Reply:
I realise now after my comments that `prolog.fcl`, the file that is edited in place, is actually included by another `fcl` file which is actually passed to `mu2e`. 

I agree with Rob that overriding the relevant fcl parameters makes more sense than editing them in place further up an include chain.

I made a similar command line utility while I was developing tracker alignment. It generates new FCL files, rather than editing existing files so there's sufficient bookkeeping.